template/build/webpack.base.conf.js

```diff
@@ -16,10 +16,12 @@ module.exports = {
     extensions: ['', '.js', '.vue'],
     fallback: [path.join(__dirname, '../node_modules')],
     alias: {
-      {{#if_eq build "standalone"}}
-      // nessessary to make "import Vue from 'vue'" load the standalone version.
+      {{#if_eq build "standalone"}}∞
       'vue': 'vue/dist/vue',
       {{/if_eq}}
```

Comment:
Won't the above 3 lines be unnecessary if the standalone version is the default export?